README.md

```diff
@@ -241,6 +245,7 @@ This is only required on Cloud TPU VMs.
 
 | Version | GPU CUDA 11.8 + Docker |
 | --- | ----------- |
+| 2.2 | `us-central1-docker.pkg.dev/tpu-pytorch-releases/docker/xla:r2.2.0_3.10_cuda_11.8` |
```

Comment:
we seems don't have the 2.2.0 image for cuda11.8: https://github.com/pytorch/xla/blob/master/infra/tpu-pytorch-releases/artifacts.auto.tfvars

Reply:
Thanks for catching that, removed this for now.